test/stage1/behavior/atomics.zig

```diff
@@ -161,3 +165,54 @@ fn testAtomicRmwFloat() void {
     _ = @atomicRmw(f32, &x, .Sub, 2, .SeqCst);
     expect(x == 4);
 }
+
+test "atomicrmw with ints" {
+    if (builtin.arch == .mipsel)
+        return error.SkipZigTest;
```

Comment:
Do we have a ticket for this?

Reply:
There is #3345 but for this specific case there is no issue.

Reply:
Feel free to open a placeholder issue so that you can link to it here, annotating why it is being skipped